package.json

```diff
@@ -13,27 +13,43 @@
     "test:lint": "standard | snazzy",
     "test:unit": "tap test/unit/*_test.js",
     "test:integration": "tap test/integration/*_test.js",
-    "test": "npm run test:lint && npm run test:unit && npm run test:integration",
-    "coverage": "npm run test:unit -- --cov",
-    "lint": "standard --fix"
+    "test": "yarn run test:lint && yarn run test:unit && yarn run test:integration",
+    "coverage": "yarn run test:unit -- --cov",
+    "lint": "standard --fix",
+    "pack": "cross-env NODE_ENV=production electron-builder --publish=never"
   },
   "build": {
     "appId": "com.plotly.image-exporter.graph-exporter",
-    "electronCompile": false,
     "productName": "Plotly Graph Exporter",
-    "copyright": "Copyright © 2018 ${author}",
+    "files": [
+      "bin",
+      "src"
+    ],
     "appImage": {
-      "systemIntegration": "doNotAsk",
-      "category": "Utility"
+      "systemIntegration": "doNotAsk"
+    },
+    "linux": {
+      "category": "Utility",
+      "executableName": "plotly-graph-exporter",
+      "maintainer": "enquiries@nicolasriesco.net",
```

Comment:
@etpinard I guess that here should go your email.

Reply:
@n-riesco Please excuse this error or mine! I am sure I copied this from Falcon.

@etpinard @jackparmer @chriddyp Would you like me to put one of my email address in here? I suspect you'd prefer something which looks like `support@plotly.com`, and I probably would, too, but let me know.

Reply:
you can put `chris@plot.ly` if you want

Reply:
I have done so.